dts/bindings/sensor/phosense,xbr818.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2024 MASSDRIVER EI (massdriver.space)
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    Phosense XBR818 I2C-capable 10Ghz Radar Sensor
+
+compatible: "phosense,xbr818"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+  int-gpios:
+    type: phandle-array
+    required: true
+    description: GPIO pin connected to IO pin (IO_VAL)
+
+  i2c_en-gpios:
```

Comment:
cc @decsny Look this, I think we shouldn't use two styles of delimiters, WDYT?
```suggestion
  i2c-en-gpios:
```

Reply:
It's not delimiting anything here, it's the name for this pin (i2c_en, iic_en or iien depending on the specific board)